src/http/client.rs

```diff
@@ -111,6 +111,20 @@ impl AybClient {
             .await
     }
 
+    pub async fn list_databases(&self, entity: &str) -> Result<EntityQueryResponse, AybError> {
```

Comment:
Discussion: the client calls it `list`, but the server calls it `entity_details`/`query_entity`. I get the sense we'll add other details to the endpoint in the future (e.g., name, created at, etc.), so do we want to make the client consistent with that (that it's more of a `details` endpoint than a `list`ing endpoint)? I don't feel strongly now about naming, so this is more so about consistency between client and server naming.